statsmodels/tsa/statespace/mlemodel.py

```diff
@@ -482,6 +482,29 @@ def fit(self, start_params=None, transformed=True,
 
             return res
 
+    @property
+    def _res_classes(self):
+        return {'fit': (MLEResults, MLEResultsWrapper)}
+
+    def _wrap_results(self, params, result, return_raw, cov_type=None,
+                      cov_kwds=None, results_class=None, wrapper_class=None):
+        if not return_raw:
+            # Wrap in a results object
+            result_kwargs = {}
+            if cov_type is not None:
+                result_kwargs['cov_type'] = cov_type
+            if cov_kwds is not None:
+                result_kwargs['cov_kwds'] = cov_kwds
+
+            if results_class is None:
+                results_class = self._res_classes['fit'][0]
```

Comment:
Why use an opaque tuple?  Why not something that would provide meaning like a nested dict or named tuple?

Reply:
Brevity I guess.  This usage is an outlier.  Other places I've implemented it just become e.g. `(res_class, wrap_class) = self._res_classes['fit_regularized']`.  Have a particular preference for what it should look like?